ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -46,12 +46,6 @@ import UIKit
     var animationDurationForHide: TimeInterval { return Constants.animationDurationForHide }
     var animationDampingRatio: CGFloat { return isToast ? Constants.animationDampingRatioForToast : 1 }
 
-    var needsFullWidth: Bool { return !isToast }
-    var needsSeparator: Bool { return  self == .primaryOutlineBar }
-    var supportsTitle: Bool { return isToast }
-    var supportsImage: Bool { return isToast }
```

Comment:
why are these other 4 things deleted?

Reply:
I suspect because they're not actually consumed anywhere (just checked in code).

Reply:
Right, they were used in the old UIKit one, but aren't being used anywhere, so I figured the clean up just got missed in the transition.

Reply:
should be part of the description of the pr

Reply:
Fortunately these were never public APIs, so there's nothing breaking about it.